react-rendering-lists/src/index.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+const pokedex = [
+  { number: '001', name: 'Bulbasaur' },
+  { number: '004', name: 'Charmander' },
+  { number: '007', name: 'Squirtle' },
+  { number: '025', name: 'Pikachu' },
+  { number: '039', name: 'Jigglypuff' }
+];
+
+class PokeList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.pokemonList = props.pokemonList;
```

Comment:
You almost never want to store your props. Instead, just refer to `this.props` in your `render()` method.